### PR TITLE
add flakehell exception to one file pyproject.toml

### PR DESCRIPTION
**Pull request by bandersen23**
I think this should fix the code quality error by flakehell. 👍

### CODE REVIEW

**Comment by pawamoy** [comment]
Oh, thanks, but could add a `# noqa: WPS430` comment next to the nested function definition instead? I don't want to disable it for the whole file. Thanks!

**Comment by bandersen23** [comment]
Correction, not fixed. Will look at it more later. My apologies.

File: src/mkdocstrings/loggers.py [hunk 1]
```diff
@@ -62,7 +62,7 @@ def __init__(self, logger: LoggerAdapter):
         self.critical = get_template_logger_function(logger.critical)
 
 
-def get_template_logger_function(logger_func: Callable) -> Callable:
+def get_template_logger_function(logger_func: Callable) -> Callable: # noqa: WPS430 (nested function)
```
Comment (pawamoy) [inline src/mkdocstrings/loggers.py:65]:
```suggestion
def get_template_logger_function(logger_func: Callable) -> Callable:  # noqa: WPS430 (nested function)
```

**Comment by pawamoy** [comment]
Done, thanks again!

**Pull request merged**